.github/workflows/pytest.yml

```diff
@@ -172,19 +173,36 @@ jobs:
     defaults:
       run:
         shell: bash -l {0}
-
     steps:
+      - name: Collect Workflow Telemetry
+        uses: catchpoint/workflow-telemetry-action@v2
+
+      - name: Show disk space usage 1
+        run: |
+          df -h
+          du -sh /home/runner/*
```

Comment:
I probably went overboard with these.  Also, disk isn't our limiting factor right now, so we can probably cut most of these.

Reply:
I think these are useful! I would keep 3, 5, and 6. 1, 2, and 4 don't really seem actionable to me...